go.mod

```diff
@@ -2,11 +2,6 @@ module github.com/grafana/grafana
 
 go 1.22.4
 
-// Override docker/docker to avoid:
-// go: github.com/drone-runners/drone-runner-docker@v1.8.2 requires
-// github.com/docker/docker@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
-replace github.com/docker/docker => github.com/moby/moby v23.0.4+incompatible
```

Comment:
@kminehart sorry for the ping but since we have merged https://github.com/grafana/grafana/pull/89243 I believe we can remove this right? It is causing some issues for me